test/dspal_tester/adsp_proc/gpio_test_imp.c

```diff
@@ -216,6 +217,100 @@ int dspal_tester_test_gpio_read_write(void)
 	return result;
 }
 
+/**
```

Comment:
The documentation for DSPAL tester (not just DOXYGEN docs) need to explain the HW setup needed to run this test.